source/include/client.h

```diff
@@ -74,39 +74,39 @@ LIBRARY_API const char* udaGetBuildVersion();
  */
 LIBRARY_API const char* udaGetBuildDate();
 
-LIBRARY_API const char* getIdamServerHost();
+LIBRARY_API const char* udaGetServerHost(); 
 
-LIBRARY_API int getIdamServerPort();
+LIBRARY_API int udaGetServerPort(); 
```

Comment:
Is the whitespace after all these functions intended? Or something clang-format did?

Reply:
Think from a lazy copy and paste in some find/replace regex. I'd hoped clang-format would remove it.